op3_ball_detector/include/op3_ball_detector/ball_detector.h

```diff
@@ -84,10 +85,15 @@ class BallDetector
   bool getParamCallback(op3_ball_detector::GetParameters::Request &req, op3_ball_detector::GetParameters::Response &res);
 
   bool saveImageCallback(op3_ball_detector::SaveImage::Request &req, op3_ball_detector::SaveImage::Response &res);
+  bool switchDetectionCallback(op3_ball_detector::SwitchDetection::Request &req, op3_ball_detector::SwitchDetection::Response &res);
+
+  bool loadDetectionSettings();
+  void applyDetectionSettings(); // types TBD
 
   void resetParameter();
   void publishParam();
 
+
```

Comment:
This line should be removed if we can squish that in with whichever commit added it, if not it's ok with me to merge